pkg/controller/instance/instance_controller.go

```diff
@@ -186,30 +174,21 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		return reconcile.Result{}, err // OV not found has to be retried because it can really have been created after Instance
 	}
 
-	// ---------- 2. Check if we should start execution of new plan ----------
+	// ---------- 2. Execute scheduled plan if exists ----------
+	ensurePlanStatusInitialized(instance, ov)
 
-	newExecutionPlan, err := newExecutionPlan(instance, ov)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-
-	if newExecutionPlan != nil {
-		log.Printf("InstanceController: Going to start execution of plan '%s' on instance %s/%s", convert.StringValue(newExecutionPlan), instance.Namespace, instance.Name)
-		err = startPlanExecution(instance, convert.StringValue(newExecutionPlan), ov)
-		if err != nil {
-			return reconcile.Result{}, r.handleError(err, instance, oldInstance)
-		}
-		r.Recorder.Event(instance, "Normal", "PlanStarted", fmt.Sprintf("Execution of plan %s started", convert.StringValue(newExecutionPlan)))
```

Comment:
Let's keep these events and the `log.Printf` above for a good debugging experience. How about logging this after line 185, when it's clear that we have a plan scheduled?

Reply:
Hm, we don't make this distinction between a _new_ and _existing_ plan anymore. That's what all the removed logic was about. But I agree that this logline is useful. I'll rephrase a bit and add it back.